cer-graphql/index.js

```diff
@@ -8,12 +8,38 @@ const jwkToPem = require('jwk-to-pem');
 // Measure server startup time
 var startTime = new Date().getTime(); 
 
+/**
+ * Deletes environment variables we expect to use as existing env vars dont get overwritten.
+ */
+const deleteEnvironmentVariables = () => {
+    delete process.env.CONTENTFUL_ACCESS_TOKEN;
+    delete process.env.CONTENTFUL_SPACE_ID;
+    delete process.env.COGNITO_USER_POOL; 
+    delete process.env.COGNITO_REGION; 
+}
+
 const getCredentials = (isFromFile) => {
     // isFromFile determines where we load the credentials from.
     // If true we load from the .env file in the folder. 
     // If false, we load from environment variables.
     if (isFromFile) {
-        const configResult = require('dotenv').config({path: '../.env'});
+
+        deleteEnvironmentVariables();
+
+        let path;
+        switch(process.env.awsProfile) {
```

Comment:
where does this env variable get set?

Reply:
ah nevermind, I see it's done in package.json script

Reply:
I would rather this variable was called 'stage' or 'env', seeing it's used to refer to the correct env file. You should pass in the AWS profile as a separate variable - this is because the 'stage' and the profile won't always match up. E.g. in Jenkins, we deploy to nonprod branch (= 'test' stage), but the aws profile that is used is called 'uoa-its-nonprod' (not test).

Reply:
I'm also making mods in serverless-now similar to this because we want the tests to be able to run in Jenkins